tests/test_validators.py

```diff
@@ -22,6 +23,19 @@
 from pydantic.decorators import field_validator, root_validator
 
 
+def test_annotated_validator() -> None:
+    MyInt = Annotated[int, AfterValidator(lambda x: x if x != -1 else 0)]
```

Comment:
we should simplify tests to show using just one `AfterValidator`, then more complex usage.

Reply:
split it up into two tests